jerry-core/debugger/debugger.c

```diff
@@ -514,6 +519,10 @@ jerry_debugger_breakpoint_hit (uint8_t message_type) /**< message type */
 
   while (!jerry_debugger_receive ())
   {
+    /* timeout */
+    #ifdef __unix__
+    syscall(SYS_nanosleep, (const struct timespec[]){{0, 100}}, NULL);
```

Comment:
I think so, because jerry-libc is turned off so nanosleep is not available and combining jerry-libc with unix-libc would not be the best choice.